include/SDL3/SDL_init.h

```diff
@@ -182,6 +188,106 @@ extern SDL_DECLSPEC SDL_InitFlags SDLCALL SDL_WasInit(SDL_InitFlags flags);
  */
 extern SDL_DECLSPEC void SDLCALL SDL_Quit(void);
 
+
+/**
+ * Specify basic metadata about your app.
+ *
+ * You can optionally provide metadata about your app to SDL. This is not
+ * required, but strongly encouraged.
+ *
+ * There are several locations where SDL can make use of metadata (an "About"
+ * box in the macOS menu bar, the name of the app can be shown on some audio
+ * mixers, etc). Any piece of metadata can be left as NULL, if a specific detail
+ * doesn't make sense for the app.
+ *
+ * This function should be called as early as possible, before SDL_Init.
+ * Multiple calls to this function are allowed, but various state might not
+ * change once it has been set up with a previous call to this function.
+ *
+ * Passing a NULL removes any previous metadata.
+ *
+ * This is a simplified interface for the most important information. You can
+ * supply significantly more detailed metadata with SDL_SetAppMetadataWithProperties.
+ *
+ * \param appname The name of the application ("My Game 2: Bad Guy's Revenge!").
+ * \param appversion The version of the application ("1.0.0beta5" or a git hash, or whatever makes sense).
+ * \param appidentifier A unique string in reverse-domain format that identifies this app ("com.example.mygame2").
+ * \returns 0 on success or a negative error code on failure; call
+ *          SDL_GetError() for more information.
+ *
+ * \since This function is available since SDL 3.0.0.
+ *
+ * \threadsafety Do not call this function from two threads at once.
```

Comment:
Do we have any standardized way of saying this? e.g. "This function is not thread-safe."

Reply:
I think generally we have gone out of our way to make things thread safe, so not at the moment.  :)

We can go with "This function is not thread-safe." though.

Reply:
Fixed in 164928aff9b82ae3a43940b0dbe41307439fbaba.